multicluster/cmd/multicluster-controller/controller.go

```diff
@@ -47,6 +47,8 @@ import (
 var (
 	validationWebhooksNamePattern = "%s%santrea-mc-validating-webhook-configuration"
 	mutationWebhooksNamePattern   = "%s%santrea-mc-mutating-webhook-configuration"
+
+	setupManagerAndCertControllerFunc = setupManagerAndCertController
```

Comment:
Could we add a comment saying unit test code will change the func to set up a mock manager.

Reply:
Added then comment.